### ISSUE TITLE
SparkFun_MS5803_I2C.cpp line 89 lists wrong units

### ISSUE THREAD

**Issue by okeefer22** [sparkfun/SparkFun_MS5803-14BA_Breakout_Arduino_Library#5]
Lines 88-96 in _SparkFun_MS5803_I2C.cpp_
```
float MS5803::getPressure(precision _precision)
// Return a pressure reading units Pa.
{
 	getMeasurements(_precision);
	float pressure_reported;
	pressure_reported = _pressure_actual;
	pressure_reported = pressure_reported / 10.0f;
	return pressure_reported;
}
```

The note says it is returning in units of Pa, but the the __pressure_actual_ that is calculated in _GetMeasurements()_ is in mbar. In order to convert from mbar to Pa, you need to multiply by 100. A division by 10 (as is done here) is for a conversion from mbar to kPa.

**Comment by PaulZC** [comment]
Hi @okeefer22 ,

Thanks for reporting this!

Agreed. That comment is wrong. But - fortunately - the [code example](https://github.com/sparkfun/SparkFun_MS5803-14BA_Breakout_Arduino_Library/blob/fb6d08d44ba2315309895d7eef9a2eca37903248/examples/SparkFun_MS5803_Demo/SparkFun_MS5803_Demo.ino#L98) correctly reports the pressure as mbar.

The divide-by-10 comes from the datasheet and is the final step in converting **P** (signed int 32, int32_t) to mbar (float):

https://github.com/sparkfun/SparkFun_MS5803-14BA_Breakout_Arduino_Library/blob/fb6d08d44ba2315309895d7eef9a2eca37903248/src/SparkFun_MS5803_I2C.cpp#L155

![image](https://github.com/sparkfun/SparkFun_MS5803-14BA_Breakout_Arduino_Library/assets/5690545/2c2e9d8f-53ee-467c-b1cb-8dbbed9cafc3)

If we change the comment, I think we're all good.

Best wishes,
Paul

**Comment by PaulZC** [comment]
Resolved in v1.1.3
